frontend/ui/elements/reader_menu_order.lua

```diff
@@ -23,7 +23,6 @@ local order = {
         "highlight_options",
         "change_font",
         "hyphenation",
-        "read_timer",
```

Comment:
It does seem a touch out of place with the other rendering options. There should at least be a separator in between if it goes in the typeset menu. I didn't want to make things controversial by changing the menu around (and I didn't have time to think about it anyway) but it should be something more like

```lua
        -- rendering stuff
        "page_overlap",
        "switch_zoom_mode",
        "set_render_style",
        --sep, I don't think highlight options go with anything else really
        "highlight_options",
        --sep, font-related stuff
        "change_font",
        "floating_punctuation",
        "hyphenation",
        --sep, "reading experience" stuff
        "read_timer"
```